infrastructure/gcp/keep-test/google-functions/keep-faucet/issue-grant.js

```diff
@@ -126,12 +155,43 @@ exports.issueGrant = async (request, response) => {
               You can find us on Discord at: https://discord.gg/jqxBU4m .\n
             `)
           })
-          .on('error', (error) => {
-            console.error(
-              `Error with account grant transaction: [${error}]; URL was [${request.url}].`
-            )
-            if (!response.headersSent) {
-              response.status(500).send(`
+          .on("error", (error) => {
+            if (error == "Error: nonce too low") {
```

Comment:
This works, right? Sometimes it's `error.message` or `error.getMessage`.

Reply:
With the testing I did locally, it worked 😬 ..not sure that's the answer you're looking for though.